packages/admin-next/dashboard/src/v2-routes/users/user-invite/components/invite-user-form/invite-user-form.tsx

```diff
@@ -0,0 +1,381 @@
+import { zodResolver } from "@hookform/resolvers/zod"
+import { ArrowPath, Link, XCircle } from "@medusajs/icons"
+import { Invite } from "@medusajs/medusa"
+import {
+  Button,
+  Container,
+  Heading,
+  Input,
+  StatusBadge,
+  Table,
+  Text,
+  Tooltip,
+  clx,
+  usePrompt,
+} from "@medusajs/ui"
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  getPaginationRowModel,
+  useReactTable,
+} from "@tanstack/react-table"
+import { format } from "date-fns"
+import {
+  adminInviteKeys,
+  useAdminCustomPost,
+  useAdminDeleteInvite,
+  useAdminInvites,
+  useAdminResendInvite,
+} from "medusa-react"
+import { useMemo } from "react"
+import { useForm } from "react-hook-form"
+import { Trans, useTranslation } from "react-i18next"
+import * as zod from "zod"
+import { ActionMenu } from "../../../../../components/common/action-menu"
+import { NoRecords } from "../../../../../components/common/empty-table-content"
+import { Form } from "../../../../../components/common/form"
+import { LocalizedTablePagination } from "../../../../../components/localization/localized-table-pagination"
+import { RouteFocusModal } from "../../../../../components/route-modal"
+import { useV2Store } from "../../../../../lib/api-v2"
+import { InviteDTO } from "@medusajs/types"
+
+const InviteUserSchema = zod.object({
+  email: zod.string().email(),
+})
+
+const PAGE_SIZE = 10
+
+export const InviteUserForm = () => {
+  const { t } = useTranslation()
+
+  const form = useForm<zod.infer<typeof InviteUserSchema>>({
+    defaultValues: {
+      email: "",
+    },
+    resolver: zodResolver(InviteUserSchema),
+  })
+
+  const { invites, isLoading, isError, error } = useAdminInvites()
```

Comment:
shouldn't this come from the new lib/v2/* hooks?

Reply:
If we can reuse hooks from v1, I don't think we should add new ones. Hopefully, it will be like this for many hooks, as we will not be introducing that many breaking changes. 

However, coming from the promotions domain (which is new), I understand the concern.